src/v/iceberg/tests/BUILD

```diff
@@ -649,3 +649,19 @@ redpanda_cc_bench(
         "@seastar//:benchmark",
     ],
 )
+
```

Comment:
>iceberg/compat: Microbenchmark for evolve_schema & friends

nit: got any numbers? Might be worth posting some in the commit message. Or at least any conclusions that you've drawn

Reply:
> posting some in the commit message

ah yeah, slipped my mind. thx

Reply:
done